qa/vagrant/pom.xml

```diff
@@ -0,0 +1,326 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.elasticsearch.qa</groupId>
+        <artifactId>elasticsearch-qa</artifactId>
+        <version>.0.0-beta1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>elasticsearch-distribution-tests</artifactId>
+    <name>QA: Elasticsearch Vagrant Tests</name>
+    <description>Tests the Elasticsearch distribution artifacts on virtual
+      machines using vagrant and bats.</description>
+    <packaging>pom</packaging>
+
+    <!-- The documentation for how to run this is in ../Vagrantfile -->
```

Comment:
I'll fix the note but yeah, I meant to put the file in the root directory. When you run vagrant it walks up your directory tree looking for a Vagrantfile. Whatever it finds it uses. It'd be super weird to put one a `/` or something but otherwise this is convenient behavior because you can start vagrant from anywhere in your project.